index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Scheherazade+New&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Material+Icons" rel="stylesheet">
+    <title>African Devs</title>
+</head>
+
+<body>
+    <header>
+        <nav id="navbar">
+            <div class="nav-buttons">
```

Comment:
Please use a section tag "<section>" instead of a "<div>". This will give semantic meaning to your work and will be used to convey the structure of the document in a clear manner. 
```suggestion
            <section class="nav-buttons">
```